src/components/icon-favourite/icon-favourite.tsx

```diff
@@ -2,24 +2,31 @@ import React from "react";
 
 export type IconFavouriteProps = {
   fill?: boolean;
+  bright?: boolean;
 };
 
-export const IconFavourite = ({ fill }: IconFavouriteProps) => {
+export const IconFavourite = ({ fill, bright }: IconFavouriteProps) => {
   // This svg is a copy from public/icons/basic/icon-heart.svg.
   // If you find out it no longer matches the original file, please update it
   // It is made as inline svg to be able to change fill value from props
+  let classes = fill
+    ? "icon-favourite icon-favourite--filled"
+    : "icon-favourite";
+
+  if (bright) {
+    classes = fill ? "icon-favourite--bright-filled" : "icon-favourite--bright";
+  }
+
   return (
     <svg
       height="24"
       width="24"
-      className="icon-favourite"
+      className={classes}
       viewBox="0 0 24 24"
-      fill={fill ? "#000" : "none"}
       xmlns="http://www.w3.org/2000/svg"
     >
       <path
         d="M11.5 20L10.2675 18.921C5.89 15.1035 3 12.5858 3 9.49591C3 6.9782 5.057 5 7.675 5C9.154 5 10.5735 5.66213 11.5 6.70845C12.4265 5.66213 13.846 5 15.325 5C17.943 5 20 6.9782 20 9.49591C20 12.5858 17.11 15.1035 12.7325 18.9292L11.5 20Z"
-        stroke="black"
```

Comment:
I know that you have had some talk about this with Kasper already (though I do not know about it in detail).
But there will still be a black outline on the un-checked icon right?

Reply:
<img width="414" alt="image" src="https://user-images.githubusercontent.com/15377965/215866493-d5184066-72a6-4447-b6c4-672207ce001e.png">

<img width="375" alt="image" src="https://user-images.githubusercontent.com/15377965/215866525-e34506f0-155b-4096-99f9-48e1abc33cbc.png">

Reply:
👍